types/src/front/lib/assistant.ts

```diff
@@ -18,7 +18,8 @@ export const isModelProviderId = (
 ): providerId is ModelProviderIdType =>
   MODEL_PROVIDER_IDS.includes(providerId as ModelProviderIdType);
 
-export const ModelProviderIdCodec = ioTsEnum(MODEL_PROVIDER_IDS);
+export const ModelProviderIdCodec =
```

Comment:
NIT: can we do the same for ModelIdCodec below?

Reply:
✅ 😃